src/views/EventListing/index.js

```diff
@@ -378,6 +382,32 @@ export class EventListing extends React.Component {
         }
     }
 
+    handleTextQueryChange = async (event) => {
+        const value = event.target.value;
+        if(this.state.textSearchQuery !== value){
+            this.setState(()=>({textSearchQuery: value}))
+
+            const queryParams = this.getDefaultEventQueryParams()
+            queryParams.text = value;
+
```

Comment:
Maybe the search value string should be [trimmed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) so that it doesnt contain any unnecessary whitespaces at either end. The search value string probably shouldn't accept a string that consists of only whitespace, it seems silly to search for content according to the amount of whitespaces.

Reply:
fixed!